cst438_gradebook/src/test/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+# JAWSDB database properties for register back end
+# REMOVED DB CREDENTIALS
```

Comment:
Credentials of JAWSDB are needed here? Or set as a default? Either way, long as the credentials are stored and recognized upon login

Reply:
I'm not committing the credentials because anyone could use them to mess around with the DB